integrations/tailwindcss-cli/tests/cli.test.js

```diff
@@ -374,6 +374,37 @@ describe('Build command', () => {
     )
   })
 
+  test('postcss-import is supported by default in watch mode', async () => {
+    cleanupFile('src/test.css')
+
+    await writeInputFile('index.html', html`<div class="md:something-cool"></div>`)
+    await writeInputFile(
+      'test.css',
+      css`
+        @import 'tailwindcss/base';
+        @import 'tailwindcss/components';
+        @import 'tailwindcss/utilities';
+        @import './imported.css';
+      `
+    )
+
+    let proc = $(
+      `${EXECUTABLE} --watch --input ./src/test.css --content ./src/index.html --output ./dist/main.css`
+    )
+    await new Promise((r) => setTimeout(r, 500))
```

Comment:
I tried to listen for `proc.onStdout` instead of an arbitrary delay, but couldn't get it to work properly. Let me know if you want me to dig deeper into that.